### PR TITLE
responsiveness to landing

### LINKED ISSUE #73: mobile responsiveness for landing
Screenshot 2024-08-01 at 11 26 37 PM

make sure it looks mobile and ipad responsiveness need adjustment
also try center some text on mobile 
make sure to comment out the other components just to not have overlap as you'll see the other components break the screen

### PR DESCRIPTION
**Pull request by zalam007**

### CODE REVIEW

**Comment by zalam007** [comment]
image
image
image

On the ipad size the navigation needs to change from text to the 3 line bars.

File: src/components/Landing.jsx [hunk 1]
```diff
@@ -21,19 +21,26 @@ const Landing = () => {
         className="object-cover w-full h-full absolute right-[10%] bottom-[20%] rotate-12 scale-125"
       />
 
-      <div className="absolute flex flex-col bottom-[6%] right-[3%] items-center  text-white">
-        <div className="flex flex-col items-start gap-3">
-          <h1 className="text-4xl  ml-6">EMPOWERING FUTURE INNOVATORS</h1>
+      <div className="absolute inset-0 flex flex-col justify-end items-center md:items-end md:pb-[6%] md:pr-[3%] text-white p-4">
+        <div className="flex flex-col items-center md:items-start gap-3 max-w-xl w-full">
```
Comment (stormyy00) [inline src/components/Landing.jsx]: add whitespace-nowrap so it appears in one line might have to move the entire container to the left more

**Comment by zalam007** [comment]
image
image
image

**Review by stormyy00 — COMMENTED** [review]
acc lets have whitespace no wrap for lg: and anything smaller can be whitespace-normal

**Review by stormyy00 — APPROVED** [review]

**Pull request merged**